data/campaigns/Eastern_Invasion/scenarios/18_Epilogue.cfg

```diff
@@ -4,282 +4,650 @@
+                            speaker,image=Konrad,portraits/konrad_II.webp
+                            message= _ "As for you, <i>necromancer</i>. You shall surrender that accursed stave to be destroyed, and are hereby forbidden to practice magic in any form, at penalty of death. Be grateful for Our mercy."
+                        [/message]
+                    [/else]
+                [/if]
                 [message]
-                    speaker=Konrad II
-                    message= _ "Engineer, much of Estmark region stands in ruins. The crown has need of your skills to clear the wreckage and organize the reconstruction."
-                [/message]
-                [message]
-                    speaker=Engineer
-                    message= _ "I’ll be glad ta help yah. It’ll be fun to work in peace again." # wmllint: no spellcheck
+                    speaker=$plague_staff_wielder_id
+                    message= _ "As you command, your Majesty."
                 [/message]
             [/then]
         [/if]
 
-        [message]
-            speaker=Konrad II
-            message= _ "Now as for you..."
-        [/message]
-
+        #------------------
+        # OWAEC AND GWEDDRY
+        #------------------
         [if]
             [have_unit]
                 id=Owaec
             [/have_unit]
+
             [then]
+                [delay]
+                    time=750
+                [/delay]
+
                 [message]
-                    speaker=Gweddry
-                    message= _ "Sire, we but did our duty."
+                    speaker,image=Konrad,portraits/konrad_II.webp
+                    message= _ "Owaec, last Prince of the Plains. You and your people have suffered much."
                 [/message]
-
                 [message]
-                    speaker=Konrad II
-                    message= _ "Quite. Now do not interrupt while I am doing mine... Kneel, Owaec."
+                    speaker=Owaec
+                    message= _ "My Lord, the clansmen gladly gave their lives for Wesnoth, and know that I would do the same without hesitation."
+                [/message]
+                [message]
+                    speaker,image=Konrad,portraits/konrad_II.webp
+                    message= _ "The true test is not to die for one’s King, but to live for Him... Kneel, Owaec."
                 [/message]
-
-                {MOVE_UNIT id=Owaec 10 8}
-
                 [message]
-                    speaker=Konrad II
-                    message= _ "Arise, Owaec, Knight Commander of the King’s Horse. And take as Our first command to you your own wish; go from here, secure our borders, and show all that the might of Wesnoth will endure."
+                    speaker,image=Konrad,portraits/konrad_II.webp
+                    message= _ "And arise, Owaec, High Lord of the Horse Clans! Your people have been shattered, but We trust none more than you to make them whole again. Ride from here, and recover in peace."
```

Comment:
Is it not appropriate for a Lord to be addressed as 'Sir'? Or is there a distinction between a knight and a horseman of the Clans?

Reply:
I don't understand what you're suggesting here…?

Reply:
Maybe it's just bad media portrayals of knighting, but I read this as Konrad knighting Owaec, and then Gweddry below. And certainly the monarch would not need to use honorifics when addressing his subordinates, but from those media portrayals I'm accustomed to the monarch subsequently saying, 'arise, Sir <knight>'.

Reply:
Ah, I get it. It's true that the monarch normally wouldn't need to use honorifics for his subordinates, but I don't think that means he never would. Particularly in a ceremonial situation it makes sense to include them.

I think it does make sense to address him as "Sir Owaec" if that's the context here.